DB tests fail on master

```smacker in ~/Work/metadata-retrieval on master
$ PSQL_USER=smacker PSQL_DB=test go test ./...
?   	github.com/src-d/metadata-retrieval/database	[no test files]
?   	github.com/src-d/metadata-retrieval/examples/cmd	[no test files]
?   	github.com/src-d/metadata-retrieval/examples/cmd/testing	[no test files]
--- FAIL: TestDownloaderTestSuite (6.59s)
    --- FAIL: TestDownloaderTestSuite/TestOfflineOrganizationDownloadWithDB (0.10s)
        --- FAIL: TestDownloaderTestSuite/TestOfflineOrganizationDownloadWithDB/src-d (0.06s)
            downloader_test.go:312:
                	Error Trace:	downloader_test.go:312
                	            				downloader_test.go:455
                	Error:      	Not equal:
                	            	expected: "2015-10-14 17:13:24 +0000 UTC"
                	            	actual  : "2015-10-14 19:13:24 +0200 CEST"

                	            	Diff:
                	            	--- Expected
                	            	+++ Actual
                	            	@@ -1 +1 @@
                	            	-2015-10-14 17:13:24 +0000 UTC
                	            	+2015-10-14 19:13:24 +0200 CEST
                	Test:       	TestDownloaderTestSuite/TestOfflineOrganizationDownloadWithDB/src-d
    --- FAIL: TestDownloaderTestSuite/TestOfflineRepositoryDownloadWithDB (5.73s)
        --- FAIL: TestDownloaderTestSuite/TestOfflineRepositoryDownloadWithDB/src-d/gitbase (5.65s)
            downloader_test.go:409:
                	Error Trace:	downloader_test.go:409
                	            				downloader_test.go:344
                	            				downloader_test.go:475
                	Error:      	Not equal:
                	            	expected: "2016-09-19 10:14:13 +0000 UTC"
                	            	actual  : "2016-09-19 12:14:13 +0200 CEST"

                	            	Diff:
                	            	--- Expected
                	            	+++ Actual
                	            	@@ -1 +1 @@
                	            	-2016-09-19 10:14:13 +0000 UTC
                	            	+2016-09-19 12:14:13 +0200 CEST
                	Test:       	TestDownloaderTestSuite/TestOfflineRepositoryDownloadWithDB/src-d/gitbase
FAIL
FAIL	github.com/src-d/metadata-retrieval/github	6.605s
?   	github.com/src-d/metadata-retrieval/github/graphql	[no test files]
?   	github.com/src-d/metadata-retrieval/github/store	[no test files]
?   	github.com/src-d/metadata-retrieval/testutils	[no test files]
ok  	github.com/src-d/metadata-retrieval/utils/ctxlog	(cached)
FAIL
```

I use local postgres instead of docker in case it matters.

//cc @kyrcha

--- a/github/downloader_test.go
+++ b/github/downloader_test.go
@@ -201,7 +201,7 @@ func testRepo(t *testing.T, oracle testutils.RepositoryTest, d *Downloader, stor
 	require.Nil(err)
 	// Sample some properties that will not change, no topics available in git-fixtures
 	require.Equal(oracle.URL, storer.Repository.URL)
-	require.Equal(oracle.CreatedAt, storer.Repository.CreatedAt.String())
+	require.Equal(oracle.CreatedAt, storer.Repository.CreatedAt.UTC().String())
 	require.Equal(oracle.IsPrivate, storer.Repository.IsPrivate)
 	require.Equal(oracle.IsArchived, storer.Repository.IsArchived)
 	require.Equal(oracle.HasWiki, storer.Repository.HasWikiEnabled)
@@ -247,7 +247,7 @@ func testOrg(t *testing.T, oracle testutils.OrganizationTest, d *Downloader, sto
 	// Sample some properties that will not change, no topics available in git-fixtures
 	require.Equal(oracle.Org, storer.Organization.Login)
 	require.Equal(oracle.URL, storer.Organization.URL)
-	require.Equal(oracle.CreatedAt, storer.Organization.CreatedAt.String())
+	require.Equal(oracle.CreatedAt, storer.Organization.CreatedAt.UTC().String())
 	require.Equal(oracle.PublicRepos, storer.Organization.PublicRepos.TotalCount)
 	require.Equal(oracle.TotalPrivateRepos, storer.Organization.TotalPrivateRepos.TotalCount)
 	require.Len(storer.Users, oracle.NumOfUsers)
@@ -331,7 +331,7 @@ func testOrgWithDB(t *testing.T, oracle testutils.OrganizationTest, d *Downloade
 	require.NoError(err, "Error in retrieving users")
 	// Checks
 	require.Equal(oracle.URL, htmlurl)
-	require.Equal(oracle.CreatedAt, createdAt.String())
+	require.Equal(oracle.CreatedAt, createdAt.UTC().String())
 	require.Equal(oracle.PublicRepos, numOfPublicRepos)
 	require.Equal(oracle.TotalPrivateRepos, numOfPrivateRepos)
 	require.Equal(oracle.NumOfUsers, numOfUsers)
@@ -428,7 +428,7 @@ func checkRepo(require *require.Assertions, db *sql.DB, oracle testutils.Reposit
 	err := db.QueryRow("select htmlurl, created_at, private, archived, has_wiki, topics from repositories where owner_login = $1 and name = $2", oracle.Owner, oracle.Repository).Scan(&htmlurl, &createdAt, &private, &archived, &hasWiki, pq.Array(&topics))
 	require.NoError(err, "Error in retrieving repo")
 	require.Equal(oracle.URL, htmlurl)
-	require.Equal(oracle.CreatedAt, createdAt.String())
+	require.Equal(oracle.CreatedAt, createdAt.UTC().String())
 	require.Equal(oracle.IsPrivate, private)
 	require.Equal(oracle.IsArchived, archived)
 	require.Equal(oracle.HasWiki, hasWiki)

